homeassistant/components/notify/xmpp.py

```diff
@@ -93,29 +118,217 @@ def __init__(self):
             if not verify_certificate:
                 self.add_event_handler('ssl_invalid_cert',
                                        self.discard_ssl_invalid_cert)
+            if data:
+                # Init XEPs for image sending
+                self.register_plugin('xep_0030')  # OOB dep
+                self.register_plugin('xep_0066')  # Out of Band Data
+                self.register_plugin('xep_0071')  # XHTML IM
+                self.register_plugin('xep_0128')  # Service Discovery
+                self.register_plugin('xep_0363')  # HTTP upload
 
             self.connect(force_starttls=self.force_starttls, use_ssl=False)
 
-        def start(self, event):
+        async def start(self, event):
             """Start the communication and sends the message."""
-            self.get_roster()
-            self.send_presence()
```

Comment:
Was removing `get_roster()` and `send_presence()` done on purpose, it seems unrelated?

Reply:
It's not needed to send a message, must have been there from sample code before. adds time and possible places for error to the request.